test/jest/Netscript/RunScript.test.ts

```diff
@@ -77,11 +77,11 @@ test.each([
   },
 ])("Netscript execution: $name", async function ({ expected: expectedLog, scripts }) {
   global.URL.createObjectURL = function (blob) {
```

Comment:
Huh, surprised you couldn't just put the type here.

Reply:
TS wants blob to be exactly `Blob | MediaSource` (signature of `global.URL.createObjectURL`). I tried `global.URL.createObjectURL = function (blob: { code: string }) {`, but TS does not allow doing that without `// @ts-expect-error`.